templates/r2_pos_plot.R

```diff
@@ -18,7 +18,7 @@ make_option(c("-t", "--target"), action = "store", default = "${target}", type =
 help = "Target .frq file"),
 make_option(c("-o", "--output"), action = "store", default = "${output}", type = 'character',
 help = "Output .png file"),
-make_option(c("-m", "--maf"), action="store", default = NA, type = 'character',
+make_option(c("-m", "--maf"), action="store", default = "${}", type = 'character',
```

Comment:
This should not be an issue as `--maf` is currently not used. 
I am wondering why it is throwing an error on your side.